scripts/validate-helm.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+if [ -L ${BASH_SOURCE-$0} ]; then
+  PWD=$(dirname $(readlink "${BASH_SOURCE-$0}"))
+else
+  PWD=$(dirname ${BASH_SOURCE-$0})
+fi
+export CURRENT_PATH=$(cd "${PWD}" >/dev/null; pwd)
+export KUBERAY_HOME=${CURRENT_PATH}/..
+
+# Install kubeconform
+KUBECONFORM_INSTALL=$(mktemp -d)
+trap 'rm -rf "${KUBECONFORM_INSTALL}"' EXIT
+curl -L https://github.com/yannh/kubeconform/releases/latest/download/kubeconform-linux-amd64.tar.gz | tar xz -C "${KUBECONFORM_INSTALL}"
+curl -o "${KUBECONFORM_INSTALL}/crd2schema.py" https://raw.githubusercontent.com/yannh/kubeconform/master/scripts/openapi2jsonschema.py
+SCRIPT_PATH="${KUBECONFORM_INSTALL}/crd2schema.py"
+RAYCLUSTER_CRD_PATH="$KUBERAY_HOME/ray-operator/config/crd/bases/ray.io_rayclusters.yaml"
+
+# Convert Ray Cluster CRD YAML to JSON Schema
+pushd "${KUBECONFORM_INSTALL}" > /dev/null
+python3 "$SCRIPT_PATH" "$RAYCLUSTER_CRD_PATH"
```

Comment:
Don't call `python3` directly. Call `$SCRIPT_PATH"` directly and use its she-bang instead. It is declared as `#!/usr/bin/env python3` so it is more reliable.